MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -254,13 +296,32 @@ - (void)loadCache
         NSArray *legacyTokens = [self.legacyAccessor allTokensWithContext:nil error:nil];
         [allTokens addObjectsFromArray:legacyTokens];
         
+        BOOL isPopToken = NO;
         for (MSIDBaseToken *token in allTokens)
         {
+            if ([token isKindOfClass:[MSIDAccessTokenWithAuthScheme class]])
+            {
+                isPopToken = YES;
```

Comment:
should we check actual scheme here? Not all MSIDAccessTokenWithAuthScheme will be POP tokens in the future.

Reply:
Addressed.